installer/roles/local_docker/templates/docker-compose.yml.j2

```diff
@@ -123,3 +128,14 @@ services:
       POSTGRES_DB: {{ pg_database }}
       PGDATA: /var/lib/postgresql/data/pgdata
   {% endif %}
+
+{% if additional_task_volumes is defined %}
+{% set named_volumes = additional_task_volumes | reject("match", "^[^:]*/[^:]*:[^:]*(:.*)?") | list %}
```

Comment:
Is there any better way to do this than this regex?  This is pretty hairy.

Reply:
I wanted to go with the "in" test but it's not available in the current jinja version used. It's only about finding out whether there's a slash in the outer part of the volume path. Could split it before and then work on the first part of the split. That would turn this line into a few, not sure that would make it easier.

The other option would be changing the input from the inventory to be a hash that carries more information than just the volume string.